code/modules/jobs/job_types/roguetown/church/priest.dm

```diff
@@ -228,6 +228,8 @@ GLOBAL_LIST_EMPTY(heretical_players)
 		if(!mind.has_spell(type))
 			mind.AddSpell(new type)
 
+	src.devotion.devotion *= 0.4
```

Comment:
they start with 0, so this would be multiply 0 by 0.4.

Reply:
It doesn't reset their devotion datum anymore so they keep what they have before I reduce it.